tornado-examples/src/main/java/uk/ac/manchester/tornado/examples/arrays/ArrayAddIntPrebuilt.java

```diff
@@ -62,18 +70,27 @@ public static void main(String[] args) {
         String filePath = tornadoSDK + "/examples/generated/";
         filePath += device.getPlatformName().contains("PTX") ? "add.ptx" : "add.cl";
 
+        AccessorParameters accessorParameters = new AccessorParameters(3);
+        accessorParameters.set(0, a, Access.READ_ONLY);
+        accessorParameters.set(1, b, Access.READ_ONLY);
+        accessorParameters.set(2, c, Access.WRITE_ONLY);
+
         TaskGraph taskGraph = new TaskGraph("s0") //
                 .transferToDevice(DataTransferMode.FIRST_EXECUTION, a, b) //
-                .prebuiltTask("t0", "add", filePath, //
-                        new Object[] { a, b, c }, //
-                        new Access[] { Access.READ_ONLY, Access.READ_ONLY, Access.WRITE_ONLY }, //
-                        device, //
-                        new int[] { numElements }) //
+                .prebuiltTask("t0", "add", filePath, accessorParameters) // 
                 .transferToHost(DataTransferMode.EVERY_EXECUTION, c);
 
-        ImmutableTaskGraph immutableTaskGraph = taskGraph.snapshot();
-        TornadoExecutionPlan executor = new TornadoExecutionPlan(immutableTaskGraph);
-        executor.execute();
+        WorkerGrid workerGrid = new WorkerGrid1D(numElements);
+        GridScheduler gridScheduler = new GridScheduler("s0.t0", workerGrid);
+
+        try (TornadoExecutionPlan executor = new TornadoExecutionPlan(taskGraph.snapshot())) {
+            executor.withDevice(device) //
+                    .withGridScheduler(gridScheduler) //
+                    .execute();
+
```

Comment:
Remove this line.

In this example, I get the message:
```java
a: uk.ac.manchester.tornado.api.types.arrays.IntArray@4f6f416f
b: uk.ac.manchester.tornado.api.types.arrays.IntArray@6d07a63d
c: uk.ac.manchester.tornado.api.types.arrays.IntArray@1bd39d3c
```

Do we need to print this message? or is it a leftover?

Reply:
It was like that but we can remove it.